examples/examples-cpu/prefect/prefect-scoring.ipynb

```diff
@@ -321,7 +321,7 @@
    "source": [
     "executor = DaskExecutor(\n",
     "    cluster_class=SaturnCluster,\n",
-    "    adapt_kwargs={\"minimum\": 2, \"maximum\": 2}\n",
+    "    adapt_kwargs={\"minimum\": 3, \"maximum\": 3}\n",
```

Comment:
Might be simpler to use `cluster_kwargs` and `n_workers=3`

Reply:
probably, yeah. This was written before `n_workers` played nicely with `SaturnCluster`

Reply:
sure no problem, I can update that.